calculator/main.py

```diff
@@ -58,18 +70,22 @@ def get_history(calc):
         ie:
         ('2016-05-20 12:00:00', 'add', (1, 2), 3),
     """
-    pass
+    return list(calc['history'])
```

Comment:
You don't need to explicitly cast this as a list here. You've already written your code so that the history it's always a list. (Unless of course somebody's manually tinkered with the calculator, in which case all bets are off on anything working.)